pkg/cmd/server/kubernetes/node/node_config_test.go

```diff
@@ -122,14 +120,19 @@ func TestKubeletDefaults(t *testing.T) {
 			SeccompProfileRoot:             "/var/lib/kubelet/seccomp",
 			CloudProvider:                  "auto-detect",
 			RuntimeRequestTimeout:          metav1.Duration{Duration: 2 * time.Minute},
+			ImagePullProgressDeadline:      metav1.Duration{Duration: 1 * time.Minute},
 			ContentType:                    "application/vnd.kubernetes.protobuf",
 			EnableControllerAttachDetach:   true,
+			ExperimentalQOSReserved:        componentconfig.ConfigurationMap{},
+			EnableCRI:                      true,
```

Comment:
I would probably vote to enable it. Maybe online keeps it off initially while we let CI soak it.